<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TNumberModelSettingsWidget</class>
 <widget class="QWidget" name="TNumberModelSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>236</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>225</width>
    <height>236</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="5" column="2">
    <widget class="QPushButton" name="decBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/decNumber.png</normaloff>:/Images/decNumber.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="num1Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/1number.png</normaloff>:/Images/1number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="num8Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/8number.png</normaloff>:/Images/8number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="num0Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/0number.png</normaloff>:/Images/0number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="posNegBtn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/posNegNumber.png</normaloff>:/Images/posNegNumber.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl++</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="num3Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/3number.png</normaloff>:/Images/3number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="num7Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/7number.png</normaloff>:/Images/7number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="num9Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/9number.png</normaloff>:/Images/9number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="num2Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/2number.png</normaloff>:/Images/2number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="num4Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/4number.png</normaloff>:/Images/4number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="num5Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/5number.png</normaloff>:/Images/5number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="num6Btn">
     <property name="minimumSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../images.qrc">
       <normaloff>:/Images/6number.png</normaloff>:/Images/6number.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string>6</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>num7Btn</tabstop>
  <tabstop>num8Btn</tabstop>
  <tabstop>num9Btn</tabstop>
  <tabstop>posNegBtn</tabstop>
  <tabstop>num0Btn</tabstop>
  <tabstop>decBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
